src/planner/Planner.tsx

```diff
@@ -287,6 +266,17 @@ export const Planner = (props: Props) => {
             resetKeys={[props.systemId, planner.plan, planner.blockAssets]}
         >
             <PlannerCanvas onCreateBlock={props.onCreateBlock}>
+                <svg>
+                    <defs>
+                        <g id={'svg-portal'}>
+                            <ellipse cx="25" cy="25" rx="5" ry="5" fill={'inherit'} />
+                        </g>
+                        <g id={'svg-portal-reverse'}>
+                            <ellipse cx="25" cy="25" rx="5" ry="5" fill={'inherit'} />
+                        </g>
```

Comment:
They seems to be the same?

Reply:
Removed - was left over from a more complex "portal"